types/stripe/index.d.ts

```diff
@@ -2102,6 +2102,39 @@ declare namespace Stripe {
                 success_url: string;
             }
```

Comment:
```suggestion
                /**
                 * The mode of the Checkout Session, one of payment, setup, or subscription.
                 */
                mode?: 'payment' | 'setup' | 'subscription';
            }
```

Reply:
I can close this PR I added just after yours..

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38224